xbmc/cores/VideoPlayer/VideoPlayerAudio.cpp

```diff
@@ -484,12 +484,12 @@ bool CVideoPlayerAudio::ProcessDecoderOutput(DVDAudioFrame &audioframe)
       if (!m_audioSink.Create(audioframe, m_streaminfo.codec, m_synctype == SYNC_RESAMPLE))
         CLog::Log(LOGERROR, "%s - failed to create audio renderer", __FUNCTION__);
 
-      m_audioSink.SetDynamicRangeCompression((long)(m_processInfo.GetVideoSettings().m_VolumeAmplification * 100));
-
       if (m_syncState == IDVDStreamPlayer::SYNC_INSYNC)
         m_audioSink.Resume();
     }
 
+    m_audioSink.SetDynamicRangeCompression((long)(m_processInfo.GetVideoSettings().m_VolumeAmplification * 100));
```

Comment:
Thanks for the PR. Looks sound. Whilst you are here, theres the auto diff for formatting. Can you also chang ethe c style cast (long) to a c++ static_cast<long>()

Reply:
Thank you, done!